src/jsMain/kotlin/io/beatmaps/user/account.kt

```diff
@@ -52,13 +55,16 @@ external interface AccountComponentState : RState {
     var errors: List<String>
     var usernameErrors: List<String>
     var username: String
+    var descriptionErrors: List<String>
```

Comment:
I don't understand why this is its own property. It seems to be mutually exclusive with usernameErrors